var/spack/repos/builtin/packages/py-neurodamus/package.py

```diff
@@ -13,6 +13,7 @@ class PyNeurodamus(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-py"
 
     version('develop', branch='master')
+    version('2.1.0',   tag='2.1.0')
```

Comment:
Because of some changes in this version with the interface to `CoreNEURON` there is incompatibilities with the previous versions of `neurodamus-core`. Maybe we should do something similar to the versions dependencies with `py-neurodamus` as well?

Reply:
Neurodamus-py doesn't depend on neurodamus-core/model..., because it expects a mechlib (from any model) at runtime.
But we should have runtime checks, early if possible.
What wouldn't work in previous neurodamus-py? 
We should be careful with providing compat with (maybe, at least) the previous version

Reply:
I changed this here https://bbpcode.epfl.ch/code/#/c/51154/3/hoc/Node.hoc and here https://bbpcode.epfl.ch/code/#/c/51155/8/neurodamus/node.py. So previous hoc versions get different arguments. Can we somehow check which neurodamus-core version we have then from neurodamus-py?